snudda/utils/load_network_simulation.py

```diff
@@ -161,51 +166,71 @@ def get_voltage(self, neuron_id=None):
         else:
             return voltage
 
-    """
-    def check_depolarisation_block(self, threshold=0, max_duration=100e-3):
```

Comment:
I suggest that the duration is lowered here. Some anomalies are not lasting 50 ms and would not be detected. I use 20 ms with good results

Reply:
Changed.